src/qr/dto/qr-response.ts

```diff
@@ -0,0 +1,14 @@
+export type QrResponse = {
+	'arts.id': string;
+	'arts.name': string;
+	'arts.address': string;
+	'arts.description': string;
+	'arts.created_year': number;
+	'arts.datetime_description': string;
+	'arts.closed_day_description': string;
+	'institutions.admission_fee_description': string;
+	'authors.name': string;
+	poster_path: string; // Prettierでシングルコーテーション外れる
+	glb_path: string; // Prettierでシングルコーテーション外れる
+	reality_path: string; // Prettierでシングルコーテーション外れる
```

Comment:
```suggestion
	poster_path: string;
	glb_path: string;
	reality_path: string;
```

このコメントなくて大丈夫。上の人たちが特殊なので、、、